### PR TITLE
Change Flake8 options and reduce pylint no-self-use warns for music_grabber.py

### PR DESCRIPTION
**Pull request by dashezup**
Flake8: Enable all errors and warnings except W503

> W503 (*) | line break before binary operator
> W504 (*) | line break after binary operator

https://pycodestyle.pycqa.org/en/latest/intro.html#features

W503 and W504 are conflicted and PEP 8 recommends W504 style now.

### CODE REVIEW

**Comment by dashezup** [comment]
Pushed new commit

> Pylint: reduce warns of no-self-use in music_grabber.py by changing all methods to functions

File: cpop_bot/music_grabber.py [hunk 1]
```diff
@@ -17,102 +17,101 @@ class WrongFileSizeError(ValueError):
...
+    ydl.process_info(info)
+    info = add_downloads_to_info_dict(info, ydl_opts)
+    if not files_successfully_downloaded(info['downloads']):
+        delete_downloaded_files(info['downloads'])
+        raise ValueError(webpage_info
+                         + ": failed to download audio or thumbnail")
+    return info
+
+
+def youtube_video_not_music(info):
```
Comment (konnovdev) [inline cpop_bot/music_grabber.py:55]: huh so there's no private functions inside the file in python, interesting

**Review by konnovdev — APPROVED** [review]

File: cpop_bot/music_grabber.py (hunk 1 above)
Comment (dashezup) [reply cpop_bot/music_grabber.py:55]: I think private methods `__method_name` are also not actually private in python, those are just name mangling. but I'm not sure about this.

> “Private” instance variables that cannot be accessed except from inside an object don’t exist in Python. However, there is a convention that is followed by most Python code: a name prefixed with an underscore (e.g. _spam) should be treated as a non-public part of the API (whether it is a function, a method or a data member). It should be considered an implementation detail and subject to change without notice.

https://docs.python.org/3/tutorial/classes.html#private-variables

I think we should use `_function_name` here. I'm not sure if I misunderstand anything.

Comment (dashezup) [reply cpop_bot/music_grabber.py]: Pushed new commit which does this.

**Pull request merged**